src/java/picard/cmdline/CommandLineParser.java

```diff
@@ -363,7 +395,7 @@ public void htmlUsage(final PrintStream stream, final String programName, final
         stream.println("<a id=\"" + programName + "\"/>");
```

Comment:
While you're at it (I know it's not your code, but boyscout rules apply) -- this is evil. It causes many browsers to color all the text after this in link blue until the next closing `</a>` tag.

This line should be deleted and the next line should be replaced by         

        stream.println("<h3 id=\"" + programName + "\">" + programName + "</h3>"); 

So we can still get the convenient anchor effect without the formatting wrongness. 

I made the corresponding change in the autogenerated pages on my fork's gh-pages branch, if you want to test that anchoring still works.